packages/wonder-stuff-core/src/__tests__/error-info.test.js

```diff
@@ -58,6 +58,7 @@ describe("ErrorInfo", () => {
 
             // Act
             const act = () =>
+                // $FlowIgnore[unclear-type]
```

Comment:
Turned on linting against `Object`, `any`, and `Function`. `any` is the only one where we sometimes need it, as far as I can tell - and usually just for casting things in tests or for initializing empty objects.

I think that here in the test, we could just suppress the error and not cast at all, so I'll probably change this and related locations.

Reply:
My bad.  I was looking at the line below.  
```
ErrorInfo.fromConsequenceAndCause((null: any), cause);
```
Why doesn't the arg type for `ErrorInfo.fromConsequenceAndCause` include `null`?

Reply:
Because we don't want folks to call it this way, but we have no control over how it gets called. Just because flow asserts things doesn't mean our consumer will be using flow or not have a bug that passes a bad value.